### PR TITLE
Upload and sync processed plugin system test build artifacts

### LINKED ISSUE #12968: plugins system tests build does not upload artifacts correctly
Since the processed files for plugin system tests are saved in the plugins themselves, the upload_artifacts.sh script fails when it just tries to upload the `./tests/PHPUnit/System/processed` dir.

Not sure how to fix this quickly, so creating an issue.

CC @sgiehl

### PR DESCRIPTION
**Pull request by sgiehl**
Also needs adjustments in travis-scripts and our build artifacts viewer...

fixes #12968

### CODE REVIEW

File: plugins/CoreConsole/Commands/DevelopmentSyncProcessedSystemTests.php [hunk 1]
```diff
@@ -57,10 +62,55 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+        }
+
+        $filename = sprintf('system.plugins.%s.tar.bz2', $buildNumber);
+        $urlBase  = sprintf('https://builds-artifacts.matomo.org/matomo-org/matomo/%s', $filename);
+        $tests    = Http::sendHttpRequest($urlBase, $timeout = 120);
+
+        $tarFile = $tmpDir . $filename;
+        file_put_contents($tarFile, $tests);
+
+        $tar = new Tar($tarFile, 'bz2');
```
Comment (diosmosis) [inline plugins/CoreConsole/Commands/DevelopmentSyncProcessedSystemTests.php]: Should this `$tarFile` be unlinked?

**Review by diosmosis — COMMENTED** [review]
Haven't tested, but code looks good.

**Comment by diosmosis** [comment]
@sgiehl Can you point out the PRs on travis-scripts/ui-tests-viewer (if any)?

**Comment by sgiehl** [comment]
See https://github.com/matomo-org/ui-tests-viewer/pull/32
Haven't create one for travis-ci yet: https://github.com/matomo-org/travis-scripts/compare/artifactsupload

**Pull request merged**